spec/takeaway_spec.rb

```diff
@@ -0,0 +1,57 @@
+require 'takeaway'
+require 'Order_confirmation'
+
+describe Takeaway do
+  context "Menu items" do
+    let (:takeaway) { Takeaway.new }
+
+    it "prints out the main dishes items and prices" do
+      expect(takeaway.view_menu).to eq "Samosas - £5, Ice Cream - £3, Fries - £5, Celery Juice - £5, Apple Pie - £7, Pizza - £15, Coffee - £4"
+    end
+    it "has 7 items on the menu" do
+      expect(Takeaway::MENU.count).to eq 7
+    end
+  end
+
```

Comment:
Having struggled myself beyond the 1st user story, I find your code very illuminating and very easy to follow and enjoyable to read through as you've made the process very clear! Great code!

Reply:
Thanks so much David!